{{cookiecutter.repo_name}}/devtools/conda-envs/test_env.yaml

```diff
@@ -1,20 +1,22 @@
 name: {{cookiecutter.repo_name}}-test
-channels:{% if cookiecutter.dependency_source == 'Prefer conda-forge over the default anaconda channel with pip fallback' %}
-  - conda-forge{% endif %}
+channels:{% if cookiecutter.__dependency_source == 'anaconda' %}
```

Comment:
If I'm understanding this properly, if this case happens then the channel will be set to default and won't have conda-forge? If that's the case, they won't be able to download MDAnalysis so it stops being an mdakit.

Reply:
Good catch. I had been going to move MDA to the pip downloads but hadn't yet -- firstly need to figure out how to test this 😅

Reply:
I'm not sure that offering the combination of a mixed pip + conda install is a good idea. i.e. it's what I would consider bad practice. 

AFAIK there are nearly no cases where one has something in defaults that isn't in conda-forge - why even keep it around?